daemon/daemon.go

```diff
@@ -81,6 +81,16 @@ func (d *Daemon) Run() error {
 	writeTicker := time.NewTicker(d.config.WriteInterval)
 	defer writeTicker.Stop()
 
+	var labelTicker *time.Ticker
+	if d.config.LabelRefreshInterval > 0 {
+		labelTicker = time.NewTicker(d.config.LabelRefreshInterval)
+		defer labelTicker.Stop()
+	} else {
+		// Create dummy stopped ticker if label interval is not configured
+		labelTicker = time.NewTicker(time.Duration(1) * time.Hour)
+		labelTicker.Stop()
+	}
```

Comment:
Could we maybe avoid the else by assigning the refresh interval to a placeholder?

```suggestion
	labelRefreshInterval := time.Hour
	if d.config.LabelRefreshInterval > 0 {
		labelRefreshInterval = d.config.LabelRefreshInterval
	} 
	labelTicker = time.NewTicker(labelRefreshInterval)
	defer labelTicker.Stop()
```

Reply:
I see where you are going with this. Yeah that would work too

Reply:
@pvoborni Yes I think your suggestions both make sense. I don't think it's necessary to read these values that often. When it comes to renaming, labels is indeed a bit generic of a term,  especially as it is based on prometheus terms if I'm not mistaken. For the configuration it would probably make sense to have a name that makes more sense to the user. Let's do this in a follow up as suggested.

Reply:
I've adapted to your suggestion. I like the simplicity.